recipes/omniscidb-cuda/recipe/build.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Install packages that depend on zstd (also indirectly) to resolve
+# Azure CI issue with zstd:
+conda install --yes -c conda-forge maven libarchive arrow-cpp libgdal openjdk=8 boost-cpp=1.68 cudatoolkit-dev conda
```

Comment:
This is really strange.  These should be listed in the meta.yaml

Reply:
conda install is now removed.